ui-modules/catalog/app/views/catalog/catalog.template.html

```diff
@@ -69,7 +69,7 @@
         </div>
 
         <div class="col-md-12">
-            <ul class="list-group list-group-bundles" ng-show="state.view === 'bundles'">
+            <ul class="list-group list-group-bundles" ng-if="state.view === 'bundles'">
```

Comment:
ng-if removes the whole thing from the DOM tree, I tried it - it is much slower, I would keep ng-show for performance reasons. State can be switched by click, does not make sense to modify DOM tree on every switch.

Reply:
If you have a lot of bundles with a lot of types, we _don't_ want them all to render as that is expensive on first load.  I think taking the hit when expanded/collapsed is better.

Reply:
I see, makes sense.